components/input/input.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Input</title>
+    <link rel="stylesheet" href="../../styles/documentation.css">
+    <link rel="stylesheet" href="input.css">
+    <script src="https://kit.fontawesome.com/bf13f25644.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/default.min.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/highlight.min.js"></script>
+
+</head>
+
+<body>
+    <div class="pagewrapper">
```

Comment:
Try to maintain a naming convention like camelCase or try to name your classes like this -> 'page-wrapper' or 'page_wrapper'